src/reducers/recipeReducer.js

```diff
@@ -0,0 +1,22 @@
+import { GET_RECIPE } from '../actions/action-types';
+
+const initialState = {
```

Comment:
This is not an ideal location for the initial state of the entire application

Reply:
I have added an initial state to the store.js.